File: back/src/main/java/fr/uha/ensilan/concours/back/dto/message/PlayerLeaveMessage.java
package fr.uha.ensilan.concours.back.dto.message;

import fr.uha.ensilan.concours.back.dto.user.UserDto;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PlayerLeaveMessage extends BaseMessage {
    private UserDto player;

    public PlayerLeaveMessage() {
        super(MessageType.PLAYER_LEAVE);
    }
}

File: front/src/app/core/domain/message/player-leave.message.ts
import {BaseMessage} from "@domain/message/base.message";
import {MessageTypeEnum} from "@domain/message/message-type.enum";
import {UserDto} from "@domain/user.dto";

export class PlayerLeaveMessage extends BaseMessage {
  override readonly type = MessageTypeEnum.PLAYER_LEAVE;
  player: UserDto;
}
